compressor_integration_tests/src/lib.rs

```diff
@@ -300,6 +300,24 @@ pub fn database_structure_matches_map(state_group_map: &BTreeMap<i64, StateGroup
     true
 }
 
+/// Clears the compressor state from the database
+pub fn clear_compressor_state() {
+    // connect to the database
+    let mut builder = SslConnector::builder(SslMethod::tls()).unwrap();
+    builder.set_verify(SslVerifyMode::NONE);
+    let connector = MakeTlsConnector::new(builder.build());
+
+    let mut client = Client::connect(DB_URL, connector).unwrap();
+
+    // delete all the contents from the state compressor tables
+    let sql = r"
+        DELETE FROM state_compressor_state;
+        DELETE FROM state_compressor_progress;
```

Comment:
```suggestion
        TRUNCATE state_compressor_state;
        TRUNCATE state_compressor_progress;
```

Not that it should matter, but annoyingly `TRUNCATE` is faster than a `DELETE FROM` without a where clause.